confgenerator/config.go

```diff
@@ -43,6 +43,7 @@ type UnifiedConfig struct {
 	Metrics  *Metrics  `yaml:"metrics"`
 	// FIXME: OTel uses metrics/logs/traces but we appear to be using metrics/logging/traces
 	Traces *Traces `yaml:"traces,omitempty"`
+	Global *Global `yaml:"global,omitempty"`
```

Comment:
Shall we add some testcases into https://github.com/GoogleCloudPlatform/ops-agent/tree/master/confgenerator/testdata/valid

Reply:
I am a bit confused maybe. Are we exposing this log rotation for customer to config or not? 🤔 Will there be a public doc explaining how to config it? If so, it's user visible then some testcases inside testcases will be good to have as it's straightforward in look and clear to understand right away what config works and what not.

Reply:
Ugh.. my bad. This one never gets translated into either fluentbit or otel configs. It's remaining as its service